pkg/console/operator/sync_v400.go

```diff
@@ -278,62 +278,9 @@ func getDeploymentGeneration(co *consoleOperator) int64 {
 	return deployment.Generation
 }
 
-//func secretsMatch(secretGetter coreclientv1.SecretsGetter, clientsGetter oauthclientv1.OAuthClientsGetter) bool {
-//	secret := getSecret(secretGetter)
-//	if secret == nil {
-//		return false
-//	}
-//	oauthClient := getOauthClient(clientsGetter)
-//	if oauthClient == nil {
-//		return false
-//	}
-//	return secretAndOauthMatch(secret, oauthClient)
-//}
-
-//func getSecret(secretGetter coreclientv1.SecretsGetter) (*corev1.Secret, error) {
-//	secret, err := secretGetter.Secrets(controller.TargetNamespace).Get(secretsub.Stub().Name, metav1.GetOptions{})
-//	if apierrors.IsNotFound(err) {
-//		return nil, nil
-//	}
-//	return secret, err
-//}
-//
-//func getOauthClient(clientsGetter oauthclientv1.OAuthClientsGetter) (*oauthv1.OAuthClient, error) {
-//	oauthClient, err := clientsGetter.OAuthClients().Get(oauthsub.Stub().Name, metav1.GetOptions{})
-//	if apierrors.IsNotFound(err) {
-//		return nil, nil
-//	}
-//	return oauthClient, err
-// }
-
-// update status on CR
-// pass in each of the above resources, possibly the
-// boolean for "changed" as well, and then assign a status
-// on the CR.Status.  Be sure to account for a nil return value
-// as some of our resources (oauthlient, configmap) may simply
-// not be possible to create if they lack the appropriate inputs.
-// in this case, the Status should CLEARLY indicate this to the user.
-// Once the resource is correctly created, the status should be updated
-// again.  This should be pretty simple and straightforward work.
-// update cluster operator status... i believe this
-// should be automatic so long as the CR.Status is
-// properly filled out with the appropriate values.
-func setStatus(cs v1.ConsoleStatus, svc *corev1.Service, rt *routev1.Route, cm *corev1.ConfigMap, dep *appsv1.Deployment, oa *oauthv1.OAuthClient, sec *corev1.Secret) {
-	// TODO: handle custom hosts as well
-	if rt.Spec.Host != "" {
-		cs.DefaultHostName = rt.Spec.Host
-		logrus.Printf("stats.DefaultHostName set to %v", rt.Spec.Host)
-	} else {
-		cs.DefaultHostName = ""
-		logrus.Printf("stats.DefaultHostName set to %v", "")
-	}
+// the top level config for the console
+// this needs the console.status.publicHostname set
```

Comment:
Perhaps we want external `func`s, or perhaps we want to do these updates inline.  Haven't decided.